book/src/development/adding_lints.md

```diff
@@ -186,6 +186,13 @@ cargo dev lint input.rs
 from the working copy root. With tests in place, let's have a look at
 implementing our lint now.
 
+## Getting `cargo-clippy` and `clippy-driver` to work
```

Comment:
Can this be moved to https://github.com/rust-lang/rust-clippy/blob/master/book/src/usage.md#using-clippy-without-cargo-clippy-driver?

Reply:
That looks like more of a user guide? This is only really for development

Reply:
Oh, you are right. I misunderstood it.

Question: In what situations would we use it directly during development?

Reply:
I used it for perf testing so I could do `perf record cargo-clippy`. I'm not sure whether it's required but I have some problems with `cargo dev lint`. Although I had a lot of problems with everything to get everything to work with frame pointers :D. I personally also just like having direct access to stuff, but that's not that important.